conformance/.gitignore

```diff
@@ -0,0 +1,47 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
This could probably go in a global .gitignore instead.

Reply:
> One bigger area that doesn't seem ideal is that we're checking in the exact text of each type checker's output.

I agree it's not ideal, but I don't see a good alternative. Type checkers report the same error on different lines. Unless we mandate exactly where an error is reported (which is probably not viable), there will be many differences. Plus, type checkers sometimes report additional errors or warnings unrelated to the functionality being tested. I don't see a way around "manual scoring". That's why I think it's important to keep each test file relatively small and targeted.